CHANGELOG.md

```diff
@@ -9,6 +9,63 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.11.0] - 2022-03-03
+
+## Highlights
+
+Major changes include:
+* Changes rule L030 to use `extended_capitalisation_policy` to support Pascal case **BREAKING CHANGE**
+* Fixes are no longer applied when they corrupt the parse try causing potential other issues.
```

Comment:
Actually, at present the fixes are still applied, but it does log a warning message such as "After fixes were applied, segment failed the parse() check." Future versions may increase the log level or discard the problematic fixes. As part of this change, we did find and fix several rule issues, including the infamous "white space in name" issue.

Reply:
Updated to:

> Log warnings for fixes that seem to corrupt the parse SQL as may cause incorrect fixes in other rules.